config/symbiosis.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Would you order the networks in ascending order except `mainnet`?
something like `mainnet`, `abitrum`, `avalanche`, `bsc`, ...